### ISSUE TITLE
Brave VPN Helper service doesn't auto-launch/run when connecting

### ISSUE THREAD

**Issue by stephendonner** [brave/brave-browser#29217]
## Steps to Reproduce

   1. install `1.50.99`
   2. launch Brave
   3. restart (to pick up Griffin changes)
   4. purchase/link/set up `BraveVPN`
   5. connect to `BraveVPN`
   6. press `ctrl` + `alt` + `del` to open `Task Manager`
   7. look for the `Brave VPN Helper` task

## Actual result:

(Just on my machine, apparently...)

The `brave_vpn_helper.exe` service doesn't launch when connecting to `BraveVPN`

`Connected to BraveVPN` | `Task Manager` | `Services` panel
----------------------------|------------------|------------------
![image](https://user-images.githubusercontent.com/387249/226724347-bf94d3a1-5501-47fb-bfcc-d446313e216b.png) | ![image](https://user-images.githubusercontent.com/387249/226724536-19d774c7-a6bc-4264-80a8-ef4c7149466b.png) | ![image](https://user-images.githubusercontent.com/387249/226724836-31c09981-5ecd-45dc-be79-f6f079afb50b.png)

## Expected result:

The `BraveBetaVPNService` in Services should be `Running` when connected to `BraveVPN` on Windows 10/11

`Connected to BraveVPN` | `Task Manager` | `Services` panel
-----------------------------|------------------|-----------------
![image](https://user-images.githubusercontent.com/387249/226725694-cfe8f7d6-a41e-4798-bd07-c79a2506322b.png) | ![image](https://user-images.githubusercontent.com/387249/226725375-40a3c340-e6dd-4020-8075-1f67b407dd46.png) | ![image](https://user-images.githubusercontent.com/387249/226725149-89bb7862-4eaa-44f2-8484-3a823621c701.png)

## Reproduces how often: 

100% (but seemingly only on my Windows 11 machine; Windows 10 was fine)

## Brave version (brave://version info)

Brave | 1.50.99 Chromium: 111.0.5563.64 (Official Build) beta (64-bit)
-- | --
Revision | c710e93d5b63b7095afe8c2c17df34408078439d-refs/branch-heads/5563@{#995}
OS | Windows 11 Version 21H2 (Build 22000.1696)

## Version/Channel Information:

- Can you reproduce this issue with the current release? `No`
- Can you reproduce this issue with the beta channel? `Yes`
- Can you reproduce this issue with the nightly channel? `n/a`

cc @MadhaviSeelam @GeetaSarvadnya I believe this works fine for you two @rebron @mattmcalister @spylogsster @bsclifton @simonhong

**Comment by GeetaSarvadnya** [comment]
@stephendonner Works fine on `Windows 10 x 64 - 1.50.101`

```
Brave | 1.50.101 Chromium: 111.0.5563.116 (Official Build) beta (64-bit)
-- | --
Revision | ade86b9472831e26d87fd569e04b404be01e1e2b-refs/branch-heads/5563@{#1209}
OS | Windows 10 Version 21H2 (Build 19044.2728)

```

`Connected to BraveVPN` | `Task Manager` | `Services` panel
----------------------------|------------------|------------------
![image](https://user-images.githubusercontent.com/38657976/227918963-ae39e72d-4230-4da9-8eb5-422c5fe57c5e.png) | ![image](https://user-images.githubusercontent.com/38657976/227918900-b2e536b4-5735-4772-b721-a81ae0ea9ab5.png) | ![image](https://user-images.githubusercontent.com/38657976/227919351-93c28c99-abc2-4538-a4d4-01619990440d.png)

**Comment by spylogsster** [comment]
Not able to reproduce with others https://bravesoftware.slack.com/archives/C0YL5KMA8/p1680074153098239,

**Comment by stephendonner** [comment]
> Not able to reproduce with others https://bravesoftware.slack.com/archives/C0YL5KMA8/p1680074153098239, @stephendonner still reproducible on your side?

Yes, and @MadhaviSeelam, I believe, has the same issue, now, also on `Windows 11`.

**Comment by MadhaviSeelam** [comment]
> Not able to reproduce with others https://bravesoftware.slack.com/archives/C0YL5KMA8/p1680074153098239, @stephendonner still reproducible on your side?

@spylogsster - Seem like brave_vpn_helper.exe service launch is inconsistent.  Sometimes, it gets launched and it's not, other times in Win 11 x64.  Linking couple of recordings.
  

https://user-images.githubusercontent.com/98358127/230231167-958df631-808f-4991-b344-b895b1e2e049.mp4

https://user-images.githubusercontent.com/98358127/230234630-5b9479e1-a63d-4e68-9fee-778071939a9d.mp4

**Comment by srirambv** [comment]
just curious if the `brave_vpn_helper` application is set in the install path?

**Comment by spylogsster** [comment]
> just curious if the `brave_vpn_helper` application is set in the install path?

if you mean "Installed app" list, no

**Comment by spylogsster** [comment]
> > Not able to reproduce with others https://bravesoftware.slack.com/archives/C0YL5KMA8/p1680074153098239, @stephendonner still reproducible on your side?
> 
> @spylogsster - Seem like brave_vpn_helper.exe service launch is inconsistent. Sometimes, it gets launched and it's not, other times in Win 11 x64. Linking couple of recordings.

Thanks @MadhaviSeelam, I will check this

**Comment by spylogsster** [comment]
closing to recheck later as discused in slack https://bravesoftware.slack.com/archives/C05230YQ45C/p1684174800521609?thread_ts=1680898214.087689&cid=C05230YQ45C
